assets/cubyz/biomes/swamp/pond.zig.zon

```diff
@@ -0,0 +1,25 @@
+.{
+	.properties = .{
+		.hot,
+		.wet,
+		.overgrown,
+	},
+	.minHeight = -4,
+	.maxHeight = 1,
```

Comment:
Ponds should always be below the water level.
![Screenshot at 2025-01-15 17-17-46](https://github.com/user-attachments/assets/008f69ee-ba6a-4b65-b39b-ccc04fb088eb)